src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapTheNewGeneration.java

```diff
@@ -99,132 +99,132 @@ public void vuapDefaultStateAutoplayIsMutredAfrerReplayIsNotMuted(Page page, Str
-  public void vuapResolvedStateShouldHaveLinkToFandomArticleOnImage(Page page, String slot, String videoIframeSelector) {
-    final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
-    ads.refreshPage();
-    scrollToSlot(slot, ads);
-
-    vuap.clickOnClickArea4();
-
-    final String actual = ads.switchToNewBrowserTab();
-    Assert.assertTrue(actual.equals(FANDOM_ARTICLE_WESTWORLD_LINK), "Image should point to page on fandom.");
-  }
-
-  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateCloseOasis"},
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsVuapTngDesktop")
-  public void vuapResolvedStateShouldCloseAfterTapingOnCloseButton(Page page, String slot, String videoIframeSelector) {
-    final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
-    ads.refreshPage();
-    scrollToSlot(slot, ads);
-
-    vuap.replay();
-
-    vuap.pause();
-
-    vuap.close();
-
-    VuapAssertions.verifyReplyButtonDisplayedAfterVideoClose(vuap, MAX_AUTOPLAY_MOVIE_DURATION);
-  }
-
-  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateIsNotMuteOasis"},
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsVuapTngDesktop")
-  public void vuapResolvedStateIsNotMute(Page page, String slot, String videoIframeSelector) {
-    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
-    ads.refreshPage();
-    scrollToSlot(slot, ads);
-
-    vuap.replay();
-
-    vuap.mute();
-
-    VuapAssertions.verifyVideoUnmuteAndMute(vuap);
-  }
-
-  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedEndOasis"},
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsVuapTngDesktop")
-  public void vuapResolvedShouldEnd(Page page, String slot, String videoIframeSelector) {
-    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
-    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
-    ads.refreshPage();
-    scrollToSlot(slot, ads);
-
-    vuap.replay();
-
-    VuapAssertions.verifyReplyButtonDisplayedAfterVideoEnds(vuap, MAX_AUTOPLAY_MOVIE_DURATION);
-  }
+//  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateOnSecondPageView"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapResolvedStateAppearsOnSecondPageView(Page page, String slot, String videoIframeSelector) {
+//    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    scrollToSlot(slot, ads);
+//
+//    vuap.pause();
+//    double defaultVideoHeight = vuap.getVideoHieghtWhilePaused();
+//
+//    ads.refreshPage();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)